added note to as enterprise-only (#147)
Macrometacorp/docs: docs/collections/documents/tutorials/using-spotcollections.md

When performing read operations, here you want to read the latest copy of your data in **global** order. It provides the ability to read the latest change to your data across rows, regions and continents.

:::note

> Spot Collections are currently an Enterprise-only feature.

Reviewer: ```suggestion
Spot Collections are only available on Enterprise customer accounts.
```
